cfgov/unprocessed/js/modules/util/validators.js

```diff
@@ -21,8 +21,9 @@ const typeCheckers = require( '../../modules/util/type-checkers' );
  */
 function date( field, currentStatus ) {
   const status = currentStatus || {};
+  // Date regex matches date patterns allowed in cfgov/v1/forms.py FilterableDateField
```

Comment:
We're certainly not consistent with this, but it probably makes sense to link to a regex test bed for this regex in the code comment, like is done at https://github.com/cfpb/cfgov-refresh/blob/master/config/webpack-config.js#L24
That'll make it easier to update in the future, if needed. Totally not necessary, but an idea if you feel like it.

Reply:
Good idea. I've added that now.